### DISCUSSION TITLE
Shared Place GEDCOM v7.1 suggestions

### DISCUSSION THREAD

**Discussion by ric2016** [vesta-webtrees-2-custom-modules/vesta_shared_places#179] [Ideas]
[discussion started in the [webtrees 1.x module](https://github.com/ric2016/shared_places/issues/10)]

> I’m an advocate for augmenting GEDCOM v7.1 to include “shared places” in the standard. I’m advocating for “shared place” to have a tag name of `SPLAC` which would follow the already implemented “shared note” tag of `SNOTE` which is a change from the v5.5.1 name of `NOTE`.
> 
> What headache would it cause your module to change the current value of `_LOC` to `_SPLAC` for a future release? Could this be a setup parameter to select either `_LOC` or `_SPLAC`?
> 
> The GEDCOM committee likes to include functional updates that are already used and supported by programs today and this could be used to support this important change to the GEDCOM Standard in the future.
> 
> thanks for your feedback

**Comment by ric2016** [comment]
Supporting SPLAC/_SPLAC in place of _LOC shouldn't be too complicated. If this becomes part of the standard, I expect the webtrees core to support it ultimately anyway.

Have you reached out to someone from the GEDCOM-L group (who came up with [the specification for _LOC](https://genealogy.net/GEDCOM/))? I'm not sure if it makes sense to include the entire _LOC specification, as it is, in the standard.

**Reply by Norwegian-Sardines** [reply]
Thanks for your reply.

I think this will be a “which came first?” Issue.

The GEDCOM committee will not implement an `SPLAC` structure unless an application (preferably a commercial one) has already implemented it, and webtrees will not implement `SPLAC` or _LOC in core until it becomes part of the GEDCOM specification. 

The GEDCOM committee seams to be reluctant to do some things without buy-in from applications.

I will however reach out to _LOC committee to see if they can change their tag name for a future release.

**Comment by Norwegian-Sardines** [comment]
I wish I was smart enough to understand how to write PHP code using the modules of webtrees so I could write many of the ideas for a better GEDCOM that I have, including a “shared citation” with templates for unsupported data, or alternate data entry paths similar to Census Assistant based on other source types.

**Comment by ric2016** [comment]
> unless an application (preferably a commercial one) has already implemented it

There are definitely (other) applications that have implemented at least part of the _LOC specification, so this seems to be a likely candidate for inclusion. I didn't manage to find a list of these applications though ...
